File: KnowledgeWithoutBorders/System_KWB/Autenticação.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace System_KWB
{
    class Autenticação
    {
        // Variaveis usadas para realizar a autenticação de usuário e senha
        static string Usuário;
        static string Senha;
        static string Nível;

        public static void Login(string _Usuário, string _Senha, string _Nível)
        {
            // As variaveis abaixo não podem ser iguais as que foram usadas nos parametros do método Login() já que devemos 
            // receber os valores do mesmo
            Usuário = _Usuário;
            Senha = _Senha;
            Nível = _Nível;
        }

        public static void Recuperação(string _Usuário, string _Senha)
        {
            Usuário = _Usuário;
            Senha = _Senha;
        }

        public static String Exibir_Nível()
        {
            return Nível;
        }

        public static String Exibir_Usuario()
        {
            return Usuário;
        }

        public static String Exibir_Dados()
        {
            return "Usuário: " + Usuário + "\nSenha: " + Senha;
        }
    }
}

File: OtakuFest/OtakuFest/Autenticação.vb
Public Class Autenticação

    Public Shared Usuário As String
    Public Shared Senha As String
    Public Shared Nível As String

    Public Shared Sub Login(ByVal _Usuário As String, ByVal _Senha As String, ByVal _Nível As String)
        Usuário = _Usuário
        Senha = _Senha
        Nível = _Nível
    End Sub

    Public Shared Sub Recuperação(ByVal _Usuário As String, ByVal _Senha As String)
        Usuário = _Usuário
        Senha = _Senha
    End Sub

    Public Shared Function Exibir_Usuário() As String
        Return Usuário
    End Function

    Public Shared Function Exibir_Nível() As String
        Return Nível
    End Function

    Public Shared Function Exibir_Dados() As String
        Return "Usuário: " & Usuário & Environment.NewLine & "Senha: " & Senha
    End Function













End Class
